src/components/main_screens/SearchScreen.js

```diff
@@ -1,12 +1,54 @@
-import React from "react";
+import React, { useContext, useEffect, useState } from "react";
 import { View, StyleSheet } from "react-native";
 import { THEME } from "../../constants/Theme";
 import CustomSearch from "../reusable_components/CustomSearch";
+import AlgoquantApiContext from "../../constants/ApiContext";
 
 export default function SearchScreen({ navigation }) {
+  const [searchQuery, setSearchQuery] = useState("");
+  const [searchResults, setSearchResults] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  // State variables used to access algoquant SDK API and display/ keep state of user data from database
+  const algoquantApi = useContext(AlgoquantApiContext);
+  // Filter results whenever we change the query
+  useEffect(() => {
+    queryDatabase();
+  }, [searchQuery]);
+
+  function onChangeSearch(query) {
+    setSearchQuery(query);
+  }
+
+  // Simulates an api call with an artificial loading time
+  function queryDatabase() {
```

Comment:
I dont think `queryDatabase` should be the name of this function. Its not querying the database its making an API request to get the search results